src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -1067,7 +1067,10 @@ class FunctionSegment(BaseSegment):
                     Ref("FunctionNameSegment"),
                     max_times=1,
                     min_times=1,
-                    exclude=Ref("DatePartFunctionNameSegment"),
+                    exclude=OneOf(
+                        Ref("DatePartFunctionNameSegment"),
+                        Ref("ValuesClauseSegment"),
+                    ),
```

Comment:
This prevents the values clause from accidently being parsed as a function as described in #2406